meson_options.txt

```diff
@@ -44,6 +44,7 @@ option('framework-priority-tflite', type: 'string', value: 'tensorflow-lite,nnfw
 option('framework-priority-nb', type: 'string', value: '', description: 'A comma separated prioritized list of neural network frameworks to open a .nb file')
 option('framework-priority-bin', type: 'string', value: '', description: 'A comma separated prioritized list of neural network frameworks to open a .bin file')
 option('skip-tflite-flatbuf-check', type: 'boolean', value: false, description: 'Do not check the availability of flatbuf for tensorflow-lite build. In some systems, flatbuffers\' dependency cannot be found with meson.')
+option('srnpu-alias', type: 'string', value: 'trix-engine', description: 'The alias name list of SRNPU sub-plugin. This option provides backward compatibility.')
```

Comment:
Q: changing 'srnpu-alias' to 'trix-engine-alias'?

fw name is trix-engine and srnpu is option.
then, 'trix-engine-alias' value: 'srnpu' in meson_option and the group name in ini is [trix-engine]